flake.lock

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
This is a generated file.  Anyone who doesn't know what this is can comfortably ignore it.